installation/install-jukebox.sh

```diff
@@ -68,49 +38,93 @@ _setup_logging(){
     echo "Log start: ${INSTALL_ID}"
 }
 
+# Function to log to both console and logfile
+print_lc() {
+  local message="$1"
+  echo -e "$message" | tee /dev/fd/3
+}
+
+# Function to log to logfile only
+log() {
+  local message="$1"
+  echo -e "$message"
+}
+
+# Function to run a command where the output will be logged to both console and logfile
+run_and_log() {
```

Comment:
> to both console and logfile

Should then be called `run_and_print_lc`

Reply:
You are right!